hoomd/md/IntegrationMethodTwoStep.h

```diff
@@ -120,6 +120,13 @@ class PYBIND11_EXPORT IntegrationMethodTwoStep
             {
             }
 
+        //! Calculates force which keeps paricles on manifold in RATTLE integrators
+        /*! \param timestep Current time step
+        */
+        virtual void includeRATTLEForce(uint64_t timestep)
```

Comment:
Yes, in hindsight, having a `Timestep` type would have made changing things easier in #894. If you think it is worth making the change going forward, feel free to submit a PR.

Most of the errors involving this come from branches started before #894, then running into issues after merging master up. Whether the type in the base class is `uint64_t` or `Timestep`, if code has a method `compute(unsigned int timestep)` it will still make a new method rather than overriding the base class method.

Reply:
Right, that is annoying. I'll convert my comment to an issue just so there's a record of it.